examples/patterns/accordion.html

```diff
@@ -52,14 +52,13 @@
     .querySelector(accordionContainerSelector)
     .addEventListener('click', e => {
       const target = e.target;
-      if (target.classList.contains('p-accordion__tab')) {
-        // Find any open panels within the container and close them.
-        e.currentTarget
-          .querySelectorAll('[aria-expanded=true]')
-          .forEach(element => toggle(element, false));
-        // Open the target.
-        toggle(target, true);
-      }
+      // Find any open panels within the container and close them.
+      e.currentTarget
+        .querySelectorAll('[aria-expanded=true]')
+        .forEach(element => toggle(element, false));
```

Comment:
This should only be run on tabs that aren't the target, otherwise it toggles off and immediately on again, keeping the panel open
``` javascript
if (element !== target) {
  toggle(element, false);
}
```